QueryProfiler/Program.cs

```diff
@@ -34,41 +47,6 @@ private static void GetProfile(string query)
                    case "HasCsExpression":
                        profileScheme.Has_csCounter += 1;
                        break;
-                   case "InExpression":

```

Comment:
another way you can have this function implemented, without writing almost the same line again and again, is by adding an OperatorTranslator method, that gets the Operator.Kind, and return the matching profileScheme property name
then you can have additional method that just adds 1 by the property name


some code example of how to get from property name into actual object's property
 // Get a type object that represents the Example type.
        Type examType = typeof(ProfileScheme);

        // Change the static property value.
        PropertyInfo inCounter= examType.GetProperty("InCounter");
        inCounter.SetValue(null, 76);